core-blocks/code/edit.native.js

```diff
@@ -0,0 +1,28 @@
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { PlainText } from '../../blocks';
+
+export function edit( { attributes, setAttributes, style } ) {
```

Comment:
It can be exported as `export default function ...`.

Reply:
Makes sense, done with 0edfb18.